ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/StateTransition.java

```diff
@@ -45,6 +45,7 @@
 
   public static final String ANSI_RESET = "\u001B[0m";
   public static final String ANSI_RED = "\u001B[31m";
+  public static final String ANSI_BLUE = "\u001b[34;1m";
```

Comment:
Another good issue to create would be to just add these log colors to Alogger and just pass in a setting.  this would allow us to not have to put these all over the code

Reply:
I had this idea too. I really wanna abstract away colorful printing. Can't decide when to prioritize tho.